friktion/friktion/friktion_anchor/types/order_status.py

```diff
@@ -122,10 +120,7 @@ def from_json(obj: OrderStatusJSON) -> OrderStatusKind:
         return Filled()
     if obj["kind"] == "Disabled":
         return Disabled()
-
-    # This statement is unreachable, all cases of
-    # OrderStatusJSON are already verified
-    kind = obj["kind"]  # type: ignore
```

Comment:
This part should not be removed because
`friktion_anchor/types/order_status.py:123: error: Statement is unreachable`

Reply:
hmmm how were you able to create that error? everything under `friktion_anchor` is auto-generated, and will be wiped on another release or update so I believe it may not be worth changing unless absolutely necessary.

Reply:
It is raised by mypy when executing static type checking. But if you have no control on that module, then we may think about exclude friktion_anchor/* from the checks